rust/ql/lib/codeql/rust/elements/internal/BinaryExprImpl.qll

```diff
@@ -22,5 +21,7 @@ module Impl {
    * x += y;
    * ```
    */
-  class BinaryExpr extends Generated::BinaryExpr { }
+  class BinaryExpr extends Generated::BinaryExpr {
+    override string toString() { result = "... " + this.getOperatorName() + " ..." }
```

Comment:
We should perhaps allow specifying a `toString` implementation in the `schema` so they can be generated.

Reply:
How is that easier than doing it here?

Reply:
Instead of editing many files, you can do it all in one file. We may even be able to generate pretty sensible default `toString` implementations from the `rust.ungram` file.

Reply:
Sensible default `toString`s would be nice. However, I think this would be the right place to override them; this is also where one would add additional predicates.